FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -351,13 +352,13 @@ protected function setUrls()
 
         // This will only encode parts of the URL path and leave separator itself untouched.
         $seoUrl = $shopUrl . array_reduce(explode('/', $urlPath), function ($encodedPath, $item) {
-            $encodedPath .= '/';
+                $encodedPath .= '/';
```

Comment:
Why was this line indent when two lines below it isn't? They should be on the same indent level.

Reply:
Not sure, I simply run the composer fix before pushing the changes. Will check it again. Because a similar indentation issue had causing the lint check to fail previously.

Reply:
apparently this is the same code block which caused the lint to fail previously. The lint fix does this indentation by itself.

Reply:
I do not know what issue you had with this indent. I just fixed it by reverting the indent change you made on that line ;).

Reply:
@TheKeymaster Yes I saw, weird. But did you run `composer fix`? Because if you do, you might  see that the indentation is applied again.

Reply:
@zaifastafa no on my machine nothing is "fixed".
![Selection_009](https://user-images.githubusercontent.com/20555112/65407970-664b3e80-dde4-11e9-8c14-20e53323334a.png)